test/integration/features/user_adds_imagestream_to_project.spec.js

```diff
@@ -31,9 +31,10 @@ describe('User adds an image stream to a project', () => {
         catalogPage
           .processImageStream(JSON.stringify(centosImageStream))
           .then(() => {
-            // verify we have the nodejs image stream loaded
             let imageStreamsPage = new ImageStreamsPage(project);
             imageStreamsPage.visit();
+            // TODO: this is not a good test. The output logs will just say
+            // expected false to be true. Tests should be much more explicit.
```

Comment:
I'm guess I'm not clear on what you're proposing instead?

Reply:
I'm not proposing yet, not sure, but if it fails, the error is lousy for debugging.  I'm going to leave this TODO (but eliminate as many as possible) because I think its probably the most worthy one to actually figure out a fix.